src/pages/login.page.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [emailValid, setEmailValid] = useState(false);
+  const [password, setPassword] = useState('');
+  const [passwordValid, setPasswordValid] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+    setEmailValid(true);
+
+    if (!event.target.checkValidity()) {
+      setEmailValid(false);
+    }
+  }
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+    setPasswordValid(true);
+
+    const reg = new RegExp('^(?=.*[0-9])(?=.*[a-zA-Z])(?=.+$)');
```

Comment:
Dado que essa é uma constante clássica, daquelas que não nudam de forma alguma, talvez vc pudesse deixar antes da página? Assim a página só fica com o código próprio dela mesmo. O que acha?

Reply:
Modificação feita. Coloquei no início do documento.